cmd/extract.go

```diff
@@ -37,9 +37,12 @@ func newExtractCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var extractCmd = &cobra.Command{
 		Use:   "extract",
-		Short: "Extract the stack and your Appsody project to a local directory",
-		Long: `This copies the full project, stack plus app, into a local directory
-in preparation to build the final container image.`,
+		Short: "Extract your Appsody project to a local directory.",
+		Long: `Extract the full application (the stack and your Appsody project), into a local directory, in preparation to build the final container image.
+		
+By default, your project gets extracted into your local '$HOME/.appsody/extract' directory, unless specified otherwise.`,
```

Comment:
You don't need "By default" and "unless specified otherwise", as they basically say the same thing. How about:
"Your project is extracted into your local '$HOME/.appsody/extract' directory, unless you use the `--target-dir` flag to specify a different location."